presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -53,6 +61,24 @@
         "hive.optimized-reader.enabled"})
 public class HiveClientConfig
 {
+    public enum InsertExistingPartitionsBehavior
```

Comment:
can we move it closer to the setter and getter like how other enums are defined in the class?